libs/sdk-backend-spi/src/workspace/index.ts

```diff
@@ -28,6 +28,11 @@ export interface IAnalyticalWorkspace {
      */
     getDescriptor(): Promise<IWorkspaceDescriptor>;
 
+    /**
+     * Returns parent analytical workspace when it's a child workspace, undefined otherwise.
```

Comment:
Nitpick: formulation is strange, maybe it should be `when this workspace has a parent, undefined otherwise¨?